### ISSUE TITLE
Add a plugin for scaleway

### ISSUE THREAD

**Issue by yangm97** [docker/infrakit#675]
Scaleway is one of the cheapest cloud vendors, so I think it’s worth adding support for it, at least for testing.

**Comment by thebsdbox** [comment]
Good idea, there is an API and GO SDK (https://github.com/scaleway/scaleway-cli/tree/master/pkg/api) already so implementing a PoC shouldn't be too difficult.

**Comment by askz** [comment]
Hey any updates on this?
Would be glad to help if there's some existing work in progress since I don't have that much experience with go.

**Comment by thebsdbox** [comment]
@askz Apologies for the delay, I'll set a reminder to address this next week when I'm back at my desk.

**Comment by Sh4d1** [comment]
@thebsdbox did you start yet? If not I'll be happy to start working on it. Also scaleway released their go API separated from scaleway-cli. I already started to work on implementing scaleway provider for linuxkit, so I'm kind of used to their API :)

**Comment by thebsdbox** [comment]
@Sh4d1 I've seen your `linuxkit` work ! 

Sure, I'm in both community channels if you want to catch up about it.
